Mlem/Logic/Easter/Easter Rewards.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  Easter Rewards.swift
+//  Mlem
+//
+//  Created by tht7 on 13/07/2023.
+//
+
+import Foundation
+
+typealias IconId = String
```

Comment:
I thought about it but remember that the system dynamically loads a bunch of those from the Assets, so I'm hesitant

Reply:
Hmmm I'm not sure I'm 100% on board with that, since most icon ids won't be there by design (remember that icon ids are generated from their names in the Assets, so unless we have a build script this enum will only contain the ids of icons we want to restrict)

But I've replaced it with an enum as requested, let me know if it's better